galactic/distribution.yaml

```diff
@@ -4957,23 +4957,22 @@ repositories:
     release:
       packages:
       - webots_ros2
-      - webots_ros2_abb
+      - webots_ros2_control
       - webots_ros2_core
-      - webots_ros2_demos
+      - webots_ros2_driver
       - webots_ros2_epuck
-      - webots_ros2_examples
       - webots_ros2_importer
+      - webots_ros2_mavic
       - webots_ros2_msgs
       - webots_ros2_tesla
+      - webots_ros2_tests
       - webots_ros2_tiago
       - webots_ros2_turtlebot
-      - webots_ros2_tutorials
       - webots_ros2_universal_robot
-      - webots_ros2_ur_e_description
       tags:
         release: release/galactic/{package}/{version}
-      url: https://github.com/ros2-gbp/webots_ros2-release.git
-      version: 1.0.6-2
+      url: https://github.com/cyberbotics/webots_ros2-release.git
```

Comment:
We've changed our standard release process to use the ros2-gbp by default so that we can do automated rereleases for new platforms. Did you intentionally change this back?

FYI @nuclearsandwich

Reply:
I had to change it because I was not allowed to push on the ros2-gbp repo, although I followed the instructions printed by bloom-release.

Reply:
Ahh, we just need to give you access to that. @nuclearsandwich Can you give @omichel access for this release repository. 

More info in this thread: https://discourse.ros.org/t/upcoming-changes-to-ros2-gbp-organization/21878

Reply:
In that case, it would be nice to add all these members of the Webots maintainer group: @omichel, @BenjaminHug8, @lukicdarkoo, @stefaniapedrazzi, @BenjaminDeleze and @ad-daniel.

Reply:
@omichel I have opened https://github.com/ros2-gbp/webots_ros2-release/issues/1 to track confirming the Webots team's access to the ros2-gbp repository and provided a bit of extra info for resuming the bloom release process from that repository. My recommendation is to close this PR and create a new one using bloom to test that everything is fully functional. Please let me know in the other thread if you run into any trouble and I can assist. Thanks!

Reply:
Thank you. @BenjaminHug8 will take care of this very soon.